benchmarks/MethodOfLinesPDE/MOLxPDESystemLibrary.jl

```diff
@@ -0,0 +1,202 @@
+            continue
+
+            # advection = false
+            # discuu1 = uniformupwind1(ex, ivs, nothing, N)
+            # discuu2 = uniformupwind2(ex, ivs, nothing, N)
+            # discnu1 = chebyupwind1(ex, ivs, nothing, N)
+            # discnu2 = chebyupwind2(ex, ivs, nothing, N)
+            # discs = [discuu1, discuu2, discnu1, discnu2]
+            # if "Advection" in ex.metadata
+            #     advection = true
+            #     discw1 = discweno1(ex, ivs, nothing, N)
+            #     discw2 = discweno2(ex, ivs, nothing, N)
+            #     push!(discs, discw1, discw2)
+            # end
+
+            # probs = map(discs) do disc
+            #     discretize(ex, disc, analytic = ex.analytic_func)
+            # end
+
+            # title = "Work Precision Diagram for $(ex.name), Tags: $(ex.metadata)"
+            # println("Running $title")
+            # if advection
+            #     dummy_appxsol = [nothing for i in 1:length(probs1)]
+            #     abstols = 1.0 ./ 10.0 .^ (5:8)
+            #     reltols = 1.0 ./ 10.0 .^ (1:4);
+            #     setups = [Dict(:alg => solver, :prob_choice => 1),
+            #         Dict(:alg => solver, :prob_choice => 2),
+            #         Dict(:alg => solver, :prob_choice => 3),
+            #         Dict(:alg => solver, :prob_choice => 4),
+            #         Dict(:alg => solver, :prob_choice => 5),
+            #         Dict(:alg => solver, :prob_choice => 6),]
+            #     names = ["Uniform Upwind, center_align", "Uniform Upwind, edge_align",
+            #              "Chebyshev Upwind, center_align", "Chebyshev Upwind, edge_align",
+            #              "Uniform WENO, center_align", "Uniform WENO, edge_align"];
+
+            #     wp = WorkPrecisionSet(probs, abstols, reltols, setups; names=names,
+            #         save_everystep=false, appxsol = dummy_appxsol, maxiters=Int(1e5),
+            #         numruns=10, wrap=Val(false))
+            #     plot(wp, title=title)
+            # else
+            #     dummy_appxsol = [nothing for i in 1:length(probs)]
+            #     abstols = 1.0 ./ 10.0 .^ (5:8)
+            #     reltols = 1.0 ./ 10.0 .^ (1:4);
+            #     setups = [Dict(:alg => solver, :prob_choice => 1),
+            #         Dict(:alg => solver, :prob_choice => 2),
+            #         Dict(:alg => solver, :prob_choice => 3),
+            #         Dict(:alg => solver, :prob_choice => 4),]
+            #     names = ["Uniform Upwind, center_align", "Uniform Upwind, edge_align",
+            #              "Chebyshev Upwind, center_align", "Chebyshev Upwind, edge_align"];
+
+            #     wp = WorkPrecisionSet(probs1, abstols, reltols, setups; names=names,
+            #         save_everystep=false, appxsol = dummy_appxsol, maxiters=Int(1e5),
+            #         numruns=10, wrap=Val(false))
+            #     plot(wp, title=title)
+            # end
```

Comment:
?

Reply:
This was for the nonlinear systems, was unsure if `WPSet`  works for nonlinearsys - in any case analytic won't yet. Could I leave this skeleton in for when it does?

Reply:
I guess? It's not going to merge since it's failing anyways.

Reply:
Yes, I need to change the argument signature for the analytic generation to include `p` in MOL. Doing this in the FunctionalScheme PR.